src/Java.Interop.Tools.Cecil/Java.Interop.Tools.Cecil/DirectoryAssemblyResolver.cs

```diff
@@ -187,7 +196,7 @@ public AssemblyDefinition Resolve (AssemblyNameReference reference, ReaderParame
 			AssemblyDefinition candidate = null;
 			foreach (var dir in SearchDirectories) {
 				if ((assemblyFile = SearchDirectory (name, dir)) != null) {
-					var loaded = Load (assemblyFile);
+					var loaded = Load (assemblyFile, parameters);
```

Comment:
@lewurm did you miss adding 'ReaderParameters parameters' on line 147?
also we can make use of 
`public virtual AssemblyDefinition Load (string fileName, ReaderParameters parameters = null)`
so we don't need to duplicate the Load method same for ReadAssembly.